packages/react-core/src/next/components/Wizard/Wizard.tsx

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
-import findLastIndex from 'lodash/findLastIndex';
+import findLast from 'lodash/findLast';
```

Comment:
surprised I didn't notice this before, but is there another way we can accomplish something similar without needing to rely on lodash? that's a third party dependency we are trying to divorce ourselves from.

Reply:
Yes, we can accomplish this another way. JS is supposed to natively support the function, but I don't think all browsers support `findLast` and `findLastIndex` quite yet, and we'd have to change our tsconfig to add the latest lib ES version, which we can do another time.

Reply:
This has been addressed using a combination of `reverse` & `find`, which leads to an additional iteration, but not one that should be very taxing performance-wise. Also, once `findLastIndex` is supported we can just switch to using that function and have 1 iteration again for `goToPrevStep`.

Reply:
@tlabaj This update was made yesterday. This change is outdated.